<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>351</width>
     <height>621</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select player</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnHuman">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>240</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnComputer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>240</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnHuman</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_human_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnComputer</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>btn_computer_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>343</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_human_selected()</slot>
  <slot>btn_computer_selected()</slot>
 </slots>
</ui>
